docs/app/pages/components/sections/tables/column-sorting/snippets/app.html

```diff
@@ -1,11 +1,26 @@
 <table class="table table-hover">
     <thead>
-        <tr uxColumnSorting>
+        <tr class="text-black" uxColumnSorting>
```

Comment:
This has made the sorting arrows balc as well. Instead this class should probably just be added to the <p> tags in the table headers